<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableSelectionWindow</class>
 <widget class="QMainWindow" name="TableSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>餐桌选择</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="left_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,4,0,3,3,3,0,0">
       <item>
        <widget class="QLabel" name="label_restaurant">
         <property name="text">
          <string>欢迎光临某某餐厅</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>用户：{username}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_time">
         <property name="text">
          <string>时间：0000-00-00 00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_table_btn">
         <property name="text">
          <string>添加餐桌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="del_table_btn">
         <property name="text">
          <string>删除餐桌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dish_manage_btn">
         <property name="text">
          <string>菜品管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="history_orders_btn">
         <property name="text">
          <string>历史订单</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fullscreen_btn">
         <property name="text">
          <string>进入全屏</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="separator">
      <property name="frameShape">
       <enum>QFrame::Shape::VLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scroll_area">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>441</width>
            <height>354</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
